__main__.py

```diff
@@ -0,0 +1,7 @@
+# Entry point for rtv module
```

Comment:
This is a duplicate of ``rtv/__main__.py``. Was there an intentional reason for copying this here or was it just a merge error?